packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -313,7 +341,156 @@ const ToggleMenuDrawer = React.forwardRef(
+                                                className='header__menu--back-to-ui'
+                                                type='button'
+                                                large
+                                                onClick={() => {
+                                                    setIsPreAppStore(false);
+                                                    toggleDrawer();
+                                                }}
+                                            >
+                                                <Text className='header__menu--back-to-ui-text' size='xs'>
+                                                    {localize('Back to old interface')}
+                                                </Text>
+                                                <Icon
+                                                    className='header__menu-mobile-rightarrow'
+                                                    icon='IcArrowRight'
+                                                    size={18}
+                                                    color='red'
+                                                />
+                                            </Button>
+                                        </MobileDrawer.SubHeader>
+                                    ) : (
+                                        <MobileDrawer.SubHeader
+                                            className={classNames({
+                                                'dc-mobile-drawer__subheader--hidden': is_submenu_expanded,
+                                            })}
+                                        >
+                                            {platform_switcher}
+                                        </MobileDrawer.SubHeader>
+                                    )}
+                                    <MobileDrawer.Body>
+                                        <div
+                                            className='header__menu-mobile-platform-switcher'
+                                            id='mobile_platform_switcher'
+                                        />
+                                        {is_logged_in && (
+                                            <MobileDrawer.Item>
+                                                <MenuLink
+                                                    link_to={routes.trading_hub}
+                                                    icon='IcAppstoreHome'
+                                                    text={localize('Trading Hub')}
+                                                    onClickLink={toggleDrawer}
+                                                    changeCurrentLanguage={changeCurrentLanguage}
+                                                />
+                                            </MobileDrawer.Item>
+                                        )}
+                                        <MobileDrawer.Item>
```

Comment:
`Trade` option in menu should  not be visible from `Trading hub`, `cashier` and `accounts` page.

Reply:
Removed